### PR TITLE
feat(db-api) Document features endpoints

### LINKED ISSUE #239: Database features are not documented
Create and Remote features should be present in our public documentation

### PR DESCRIPTION
**Pull request by Soulou**
Fixes #239

### CODE REVIEW

File: _database_api/databases.md [hunk 1]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
 
 --- row ---
 
+## Enable feature
```
Comment (EtienneM) [inline _database_api/databases.md:274]:
```suggestion
## Enable Feature
```

File: _database_api/databases.md [hunk 2]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+--- row ---
+
+`POST https://$DB_API_URL/api/databases/[:db_id]/features`
+
+Enable a togglable feature for the given database.
+
+Generic features are:
+
+* `force-ssl`: Enforce database configuration to accept only TLS encrypted connections from clients
+* `publicly-available`: Require `force-ssl`, make the database accessible through its URL on the Internet and not only in Scalingo infrastructure
```
Comment (EtienneM) [inline _database_api/databases.md:285]:
```suggestion
* `publicly-available`: Require `force-ssl`, make the database accessible through its URL on the internet and not only in Scalingo infrastructure
```

File: _database_api/databases.md [hunk 3]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+
+Redis-specific features are:
+
+* `redis-rdb`: Enable RDB dump persistency mode writing down on disk database data (enabled by default)
+* `redis-aof`: Enable Append-Only File persistency mode to write down all commands and prevent data loss
+* `redis-cache`: Enable cache mode for redis deployments (least used data are automatically dropped)
```
Comment (EtienneM) [inline _database_api/databases.md:291]:
```suggestion
* `redis-cache`: Enable cache mode for Redis deployments (least used data are automatically dropped)
```

File: _database_api/databases.md [hunk 4]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+
+A `status` is returned, its state can be one of the following:
+* `PENDING`: The feature is being enabled asynchronously, please poll the database resource to get updates.
```
Comment (EtienneM) [inline _database_api/databases.md:294]: Either add a dot at the end of every item in the list, or no dot on any item.

```suggestion
* `PENDING`: The feature is being enabled asynchronously, please poll the database resource to get updates
```

File: _database_api/databases.md [hunk 5]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+{
+  "name": "force-ssl",
+  "status": "PENDING",
+  "message": "force-ssl feature is being enabled"
+}
+```
+
+--- row ---
+
+## Disable database feature
```
Comment (EtienneM) [inline _database_api/databases.md:321]:
```suggestion
## Disable Database Feature
```

File: _database_api/databases.md [hunk 6]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+* `FAILED`: Something wrong happened, please reach our support to get more information
+
+||| col |||
+
+Example request
+
+```shell
+curl -H "Accept: application/json" -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $DB_BEARER_TOKEN" \
+  -X POST https://$DB_API_URL/api/databases/:database_id/features \
```
Comment (EtienneM) [inline _database_api/databases.md:305]: To match what we have in the other pages:
```suggestion
  -X POST https://$DB_API_URL/api/databases/my-db-123/features \
```

File: _database_api/databases.md [hunk 7]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+`DELETE https://$DB_API_URL/api/databases/[:db_id]/features?feature=feature_name`
+
+disable a togglable feature for the given database.
+
+||| col |||
+
+```shell
+curl -H "Accept: application/json" -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $DB_BEARER_TOKEN" \
+  -X DELETE https://$DB_API_URL/api/databases/:database_id/features?feature=force-ssl
```
Comment (EtienneM) [inline _database_api/databases.md:334]:
```suggestion
  -X DELETE https://$DB_API_URL/api/databases/my-db-123/features?feature=force-ssl
```

File: _database_api/databases.md [hunk 8]
```diff
@@ -271,6 +271,79 @@ Returns 201 Created
...
+
+--- row ---
+
+## Disable database feature
+
+--- row ---
+
+`DELETE https://$DB_API_URL/api/databases/[:db_id]/features?feature=feature_name`
+
+disable a togglable feature for the given database.
```
Comment (EtienneM) [inline _database_api/databases.md:327]:
```suggestion
Disable a togglable feature for the given database.
```

**Review by EtienneM — COMMENTED** [review]
A few picky suggestion I'm merging to approve the PR

**Review by EtienneM — APPROVED** [review]

**Comment by Soulou** [comment]
> A few picky suggestion I'm merging to approve the PR
Thanks for the improvements

**Pull request merged**